frontend/packages/dev-console/integration-tests/features/Knative/event-Sources-InstallationView.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Event Sources Installation View
+    As a user, I should be able switch between YAML and Form view to install Event Sources
+
+    
+    Background:
+        Given user is at developer perspective
+        And user has installed OpenShift Serverless Operator
+        And user has installed Knative Apache Camelk Operator
+        And user has installed Apache Kafka Operator
```

Comment:
why is this needed? Kafka is part of serverless operator itself , need to create CR for kafka

Reply:
Updated